data/human/human missions.txt

```diff
@@ -7410,3 +7410,738 @@ mission "Adelita 5"
+			choice
+				`	(Duck.)`
+
+			`	You duck, startled by the attack, only to realize she wasn't aiming at you. The rotten-toothed man lay dead on the ground, a bullet hole in his head.`
+			`	She looks to the rest of her crew, "Only my friends call me Ildy. And I have only one friend."`
+			`	"Yes, boss."`
+			`	"Got it, boss."`
+			`	"Sure thing, Cap."`
+			`	"Whatever you say, boss."`
+			`	The group of pirates melt away into the warehouse, one of them dragging the dead body, leaving you and Ildy relatively alone.`
+			`	"I'm so glad you came, <name>." Ildico puts a friendly hand on your shoulder and squeezes.`
+			`	She turns and waves toward the warehouse. "This is my temporary headquarters. I have few other safe-houses, and own part of a hangar elsewhere. Not too many ships yet, but I intend to change that soon."`
+			`	Ildico holsters her gun right next to where she keeps her dagger and continues. "But making moves is hard. Everyone wants to be on top. Which is why I need you."`
+			`	She looks you in the eye and says, "There's a rival gang right next door to us. They have a lot more firepower, but if they could be removed, we could easily double our territory.`
+			`	"I don't have the ships," she glances at her gang in the distance, "or the quality of crew, to deal with them."`
+			`	"But, I think you do. I can't afford to pay you right now, things are pretty fluid, but I'll owe you. What do you say?"`
+			choice
+				`	"Sure, I'll help."`
+				`	"Fine. But you're going to owe big when this is all over."`
+				`	"Anything for you, Ildy."`
+				`	"Not interested."`
+					goto uninterested
+
+			`	Her smile nearly lights up the dim room. "I always know I can count on you. And only you."`
+			`	"I fed the gang some false information. They have their entire fleet right now looking for some juicy merchant ships that don't actually exist, so they're not expecting a captain like you to take them on and blow them out of the sky.`
+			`	"Check out the nearby systems and look for a ship called the <npc>, that's their flagship. Kill them all, and come back here. To me."`
+				accept
+
+			label uninterested
+			`	Ildico frowns. She looks down at her feet for a moment, and when she looks back up her face is a mask of rage and hurt. "Then get out."`
+				decline
+
+			label why?
+			`	Acting defiantly in the face of so much danger, you reach for your sidearm. Unfortunately, it's in vain, because as soon as you feel the pistol grip in your hand you simultaneously feel the burning sensations of a flurry of bullets entering your body and tearing apart your insides.`
+				die
+
+	npc kill
+		government "Bounty"
+		personality heroic staying nemesis target
+		system
+			distance 1 2
+		fleet "Marauder fleet IV"
+		dialog phrase "generic hunted bounty fleet eliminated dialog"
+
+	on visit
+		dialog phrase "generic fleet bounty hunting on visit"
+		dialog "You have destroyed the fleet. You can now return to <planet> to and report the news to Ildico."
```

Comment:
```suggestion
```

Reply:
Why the removal?

Reply:
It is in `on visit`, so it can appear even if you have not, in fact, destroyed the fleet.